specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2022-10-01-preview/examples/Registry/CodeContainer/delete.json

```diff
@@ -0,0 +1,14 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-1111-2222-3333-444444444444",
+    "resourceGroupName": "testrg123",
+    "registryName": "testregistry",
+    "api-version": "2022-10-01-preview",
+    "codeName": "testContainer"
+  },
+  "responses": {
+    "200": {},
+    "202": {},
```

Comment:
>202 [](http://example.com/codeflow?start=5&length=3)

202 should come with a Location header for the long-running operation shouldn't it?